src/services/search/service/search-index.ts

```diff
@@ -22,6 +22,7 @@ export type IndexWithPermissions = {
   size_in_mb: number;
   num_subjects: number;
   num_entries: number;
+  status: 'open' | 'delete_pending';
```

Comment:
Think we might want to keep it open-ended for future states and so it's a little less strict during creation/editing:

```suggestion
  status: 'open' | 'delete_pending' | string;
```